### PR TITLE
Fill the "Shopping Tools" topic with content

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

There is no need for the Opportunities to Engage(https://docs.magento.com/user-guide/marketing/customer-engagement.html) page as all content can be displayed on the preview page Shopping Tools(https://docs.magento.com/user-guide/marketing/shopping-tools.html) 

In this pull request, I turned two pages into one, structured the content, and added several additional positions with descriptions and links

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/marketing/shopping-tools.html
- https://docs.magento.com/user-guide/marketing/customer-engagement.html

### CODE REVIEW

File: src/marketing/shopping-tools.md [hunk 1]
```diff
@@ -1,8 +1,40 @@
...
+ - title: dotdigital Engagement Cloud
+   content: Use dotdigital Engagement Cloud to automate professional, personalized communications and generate reports using data from your Magento store.
+   url: /marketing/dotdigital/engagement-cloud.html
+
+ - title: Image zoom
+   content: When zoom is activated, customers can click the main image and move the cursor around to magnify different parts of the image.
+   url: /catalog/media-browser.html
+
+ - title: Stock Availability
+   content: Helps to notify the buyer about the availability of a product. Possibly a create the **Notify me when this product is in the stock** link for every product that is out of stock.
```
Comment (leslietilling) [inline src/marketing/shopping-tools.md:31]: Markdown formatting does not work for the "tiles". COn use HTML formatting, or just simplify for aa smalll blub of text.
```suggestion
   content: It helps buyers when you notify them about the availability of a product, and can result in increased sales. You can create the "Notify me when this product is in stock" link for every product that is out of stock.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Just a minor fix/edit

File: src/marketing/shopping-tools.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/marketing/shopping-tools.md]: Thank you for your notice, @leslietilling ☀️
I have fixed this, please approve 😊

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**